src/actors/upgrade/vcs_repos/vcs_detection/check

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import sys
+import json
+
+
+def load_inputs():
+    "Load all inputs and return dictionary with all input channels."
+    inputs = {}
+    if not sys.stdin.isatty():
+        input_data = sys.stdin.read()
+        if input_data:
+            inputs = json.loads(input_data)
+    return inputs
+
+
+def store_output(channel, data):
+    "Store the data into the expected channel (inside outputs object)."
+    # TODO: create class for input/output data? of course, this particular
+    # function works just for StringList now
+    outputs[channel] = [{"value": data}]
+    return
+
+
+def log_error(msg):
+    "Log error on stderr."
+    sys.stderr.write("Error: %s\n" % msg)
+
+
+def detect_repo(fname):
+    """
+    Check whether we found VCS repository or not.
+
+    In case we detected repository, return repository type (string). Otherwise
+    return None object.
+    """
+    for repo, suffix in vcs_map.items():
+        if fname.endswith("/%s" % suffix):
+            return repo
+    return None
+
+
+in_flist = "all_local_files"
+vcs_map = {"bzr": ".bzr/branch/branch.conf",
+           "hg": ".hg/config",
+           "git": ".git/config",
+           "svn": ".svn/entries",
+           "cvs": "CVS/Root"}
+inputs = load_inputs()
+outputs = {}
+vcs = {}
```

Comment:
Optional:
```
vcs = {repo: [] for repo in ("bzr", "hg", "git", "svn", "cvs")}
```

Reply:
Yeah i am stupid, ofc "version control system" ;D

Reply:
:-D But thanks for tip how to initialise dict. I see that for the first time and really love that construction.